dev/benchmarks/tpcbench.py

```diff
@@ -91,9 +91,12 @@ def main(benchmark: str, data_path: str, query_path: str, iterations: int, outpu
                         df.explain()
 
                         if write_path is not None:
-                            output_path = f"{write_path}/q{query}"
-                            df.coalesce(1).write.mode("overwrite").parquet(output_path)
-                            print(f"Query {query} results written to {output_path}")
+                            if len(df.columns) > 0:
```

Comment:
spark complains on saving df with empty schema, this can happen for DDL statements which came across in TPC sets